app/routes/api.getGitHubRelease.$owner.$repository.tsx

```diff
@@ -0,0 +1,41 @@
+import { LoaderFunction, json } from "@remix-run/cloudflare";
+import { GitHubRelease } from "../types/githubRelease";
+
+interface CacheEntry {
+    release: GitHubRelease;
+    timestamp: number;
+}
+
+const releaseCache: { [k: string]: CacheEntry } = {};
```

Comment:
This will just get reset on every request, because on Cloudflare Pages function only serves a single request and then terminated. To actually cache the result you are supposed to use Cache API https://developers.cloudflare.com/workers/runtime-apis/cache/.

Reply:
Since sub requests are not billed as extra just the cache headers you added should be enough.